server/testserver.go

```diff
@@ -179,8 +180,11 @@ func (ts *TestServer) Start() error {
 		}
 		stopper.Stop()
 	}
-	err = ts.Server.Start(true)
-	if err != nil {
+	if err := ts.Server.Start(true); err != nil {
+		return err
+	}
+
+	if err := testutils.SetDefaultRangeReplicaNum(ts.db, 1); err != nil {
```

Comment:
The `TestServer` never gets more to replicate to anywhere, right? I think one is fine until it does.